src/app/tutorial/tutorial.component.ts

```diff
@@ -12,23 +9,84 @@ export class TutorialComponent implements OnInit {
   currentStep: any | null = null; //if null, the tutorial will not show
   steps: any[] = [];
 
-  //********************************************
-  constructor(
-    private router: Router,
-    private route: ActivatedRoute,
-  ) {}
+  videos = [
+    {
+      id: '1_8elni4dv',
+      title: 'Workspace',
+      description:
+        'This is the workspace, a sandbox where you can design the molecules you want!',
+    },
+    {
+      id: '1_gchbjezj',
+      title: 'Block Tray',
+      description:
+        'This is the block tray, an area that stores Lego-like molecule blocks. You will use them to build your molecules.',
+    },
+    {
+      id: '1_g2u2woll',
+      title: 'Structure-Function',
+      description:
+        'Can can change the block information from structure to function view at any time. Click on an individual block to switch between function and structure modes',
+    },
+    {
+      id: '1_9qo362mt',
+      title: 'Zoom',
+      description:
+        'You can Zoom your in-process molecule, or click to reset to default view.',
+    },
+    {
+      id: '1_z7evpegt',
+      title: 'Filter',
+      description: 'Filter by block type using tags',
+    },
+    {
+      id: '1_qbzk4ge6',
+      title: 'Build and Properties',
+      description:
+        "Now that you know where everything in the DMM is, let’s show you how to build your first molecule! Click (our touch) and drag a molecule onto the worksapce to get started. When you complete a molecule you will see a summary of it's properties on the left.",
+    },
+    {
+      id: '1_m6hmfn2h',
+      title: 'Graph',
+      description:
+        'When you start making block choices, the graph will update with how many molecules are still possible.',
```

Comment:
suggested rewording: "the graph will update to show the molecules that are still possible."